app/views/messages/_form.slim

```diff
@@ -5,7 +5,8 @@
   label.no-top-margin for="message_body"
     strong = f.label(:message_label_text, class: 'no-top-margin')
   span.usa-form-hint = t('helpers.label.message.dont_sent_hint') + " "
-  = link_to 'content that would violate federal rules', about_path
+  = link_to 'content that could violate federal rules.', about_path
```

Comment:
I'm unsure how I would write the syntax here as this is a sentence with part of it being text and the other page being a link. 

<img width="625" alt="screen shot 2017-03-07 at 4 18 54 pm" src="https://cloud.githubusercontent.com/assets/1449852/23678350/c990efd2-0351-11e7-822e-8a4a0dd8080e.png">